src/main/java/de/craften/plugins/rpgplus/scripting/api/AbilityModule.java

```diff
@@ -0,0 +1,122 @@
+package de.craften.plugins.rpgplus.scripting.api;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Listener;
+import org.luaj.vm2.LuaBoolean;
+import org.luaj.vm2.LuaFunction;
+import org.luaj.vm2.LuaNumber;
+import org.luaj.vm2.LuaString;
+import org.luaj.vm2.LuaTable;
+import org.luaj.vm2.LuaValue;
+import org.luaj.vm2.lib.jse.CoerceJavaToLua;
+
+import de.craften.plugins.rpgplus.RpgPlus;
+import de.craften.plugins.rpgplus.components.ability.Ability;
+import de.craften.plugins.rpgplus.components.ability.AbilityComponent;
+import de.craften.plugins.rpgplus.scripting.ScriptingModule;
+import de.craften.plugins.rpgplus.scripting.util.ScriptUtil;
+import de.craften.plugins.rpgplus.scripting.util.luaify.Luaify;
+
+public class AbilityModule extends LuaTable implements ScriptingModule, Listener {
+	
+	private final RpgPlus plugin;
+	
+	public AbilityModule(final RpgPlus plugin) {
+		this.plugin = plugin;
+		
+		Luaify.convertInPlace(this);
+	}
+
+	@de.craften.plugins.rpgplus.scripting.util.luaify.LuaFunction("registerAbility")

```

Comment:
Please import this, the loooong line is annoying. :roll_eyes:

Reply:
The problem is it is not possible to import the LuaFunction annotation, because the class LuaFunction class of the lua-api is also needed in this class. :confused:

Reply:
:cry: